src/main/kotlin/no/nav/familie/ef/sak/journalføring/JournalføringService.kt

```diff
@@ -107,8 +110,11 @@ class JournalføringService(private val journalpostClient: JournalpostClient,
         knyttJournalpostTilBehandling(journalpost, behandling)
         grunnlagsdataService.opprettGrunnlagsdata(behandling.id)
 
-        oppdaterJournalpost(journalpost, journalføringRequest.dokumentTitler, fagsak.eksternId.id, saksbehandler)
-        ferdigstillJournalføring(journalpostId, journalføringRequest.journalførendeEnhet, saksbehandler)
+        if (journalpost.journalstatus != Journalstatus.JOURNALFOERT) {
+            oppdaterJournalpost(journalpost, journalføringRequest.dokumentTitler, fagsak.eksternId.id, saksbehandler)
+            ferdigstillJournalføring(journalpostId, journalføringRequest.journalførendeEnhet, saksbehandler)
+        }
+
```

Comment:
I hvilke tilfeller skjer dette? Burde det samme gjøres for metoden ovenfor også? 
 `journalførSøknadTilEksisterendeBehandling`

Reply:
Ja, det burde vi nok kanskje :)

Reply:
Denne situasjonen oppstår hvis man får journalført, men ikke f.eks. ferdigstilt oppgaven eller opprettet en ny oppgave. Det er ikke noe transaksjon rundt disse tingene så vi får ikke rullet tilbake.